client/src/components/button/Button.jsx

```diff
@@ -0,0 +1,32 @@
+import React, {
+    Component
+  } from 'react';
+  import '../../App.css';
+  import './button.css'
+  
+  
+  
+  class Button extends Component {
+    constructor(props) {
```

Comment:
If the constructor is empty, then you don't need a constructor, a constructor may contain either `state` or function `bind`. If you don't know what that is you can ask someone from the space.